redirects-list.json

```diff
@@ -1,7 +1,7 @@
 [
   "^https://(?:www\\.)dvc\\.org/(.+)?       https://dvc.org/$1",
   "^https://man\\.dvc\\.org/(.+)?           https://dvc.org/doc/command-reference/$1 303",
-  "^https://error\\.dvc\\.org/(.+)?         https://dvc.org/user-guide/troubleshooting#$1 303",
+  "^https://error\\.dvc\\.org/(.+)?         https://dvc.org/doc/user-guide/troubleshooting#$1 303",
```

Comment:
good catch!

Reply:
Oops. Maybe we should try to thoroughly test redirects manually (sanity check) even if we have tests because manually you can come up with ways to try to break the logic 😋  This is because publishing incorrect redirects (especially 301) causes almost permanent "damage" to new users browser cache. Just a thought

Reply:
> Needless to say I'm available to do this but just need to prioritize as I've been struggling with switching from task to task recently.